assignment/assignment-impl/impl/src/java/org/sakaiproject/assignment/impl/DownloadEventExporter.java

```diff
@@ -89,8 +89,7 @@ public void write(OutputStream output) throws IOException {
             }
             writeRow(writer, row.toArray(new String[]{}));
         });
-        writer.close();
-
+        writer.flush();
```

Comment:
Where is it closed now?

Reply:
This writer isn't, but we don't want to because it closes the enclosing OutputStream. The OutputStream will get closed as part of the normal request cycle.